pkg/controllers/csi/provisioner/install.go

```diff
@@ -56,17 +56,17 @@ func (provisioner *OneAgentProvisioner) getInstaller(ctx context.Context, dk dyn
 			Arch:          arch.Arch,
 			Flavor:        arch.Flavor,
 			Technologies:  []string{"all"},
-			TargetVersion: dk.OneAgent().GetCodeModulesVersion(),
+			TargetVersion: dk.OneAgent().GetCustomCodeModulesVersion(),
 			SkipMetadata:  true,
 			PathResolver:  provisioner.path,
 		}
 
 		urlInstaller := provisioner.urlInstallerBuilder(provisioner.fs, dtc, props)
 
 		return urlInstaller, nil
-	case dk.OneAgent().GetCodeModulesImage() != "":
+	case dk.OneAgent().GetCustomCodeModulesImage() != "":
 		props := &image.Properties{
-			ImageUri:     dk.OneAgent().GetCodeModulesImage(),
+			ImageUri:     dk.OneAgent().GetCustomCodeModulesImage(),
```

Comment:
This change is not necessary and may break things, the difference between the 2 calls is one reads from the status, the other is from the spec.

The spec one is "processed" by the version reconciler and adds the correct one in the status.

Reply:
reverted in 66eca33a7